src/main/java/seedu/address/logic/parser/ScheduleCommandParser.java

```diff
@@ -31,9 +32,9 @@ public ScheduleCommand parse(String args) throws ParseException {
         case "add":
             return new AddScheduleCommandParser().parse(arguments);
         /*case "edit":
-            return new EditScheduleCommandParser().parse(arguments);
+            return new EditScheduleCommandParser().parse(arguments);*/
         case "delete":
-            return new DeleteScheduleCommandParser().parse(arguments);*/
+            return new DeleteScheduleCommandParser().parse(arguments);
```

Comment:
Weird how this is flagged out as a warning when there exists a `DeleteScheduleCommandParserTest` file already.

Reply:
It is indeed weird that this is being flagged. 🤔